### PR TITLE
Feature/pok 32 bulk column

### PR DESCRIPTION
**Pull request by R4ngered**

### CODE REVIEW

File: PokeOneWeb/Frontend/pokeoneweb/angular.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "cli": {
+    "analytics": false
```
Comment (Finrod-Amandil) [inline PokeOneWeb/Frontend/pokeoneweb/angular.json]: What does this do?

File: PokeOneWeb/Frontend/pokeoneweb/src/app/pages/pokemon-list/pokemon-list.component.html [hunk 2]
```diff
@@ -1,973 +1,997 @@
...
+                    class="table-header-sort">⮟</div>
+            </th>
+            <th class="col-stat stat-bulk">
+                <div 
+                    [ngClass]="getSortButtonClass(column.Bulk, 1)" 
+                    (click)="sort(column.Bulk, 1)"
+                    class="table-header-sort">⮝</div>
+                <div class="has-tooltip">
+                    <div class="table-header-label">Bulk</div>
+                    <span class="tooltip">Bulk can be calculated by adding up the Healthpoints(HP), Defense(DEF) and Special Defense(SPD) stats. It can be used to search for Pokémon which are hard to defeat.</span>
```
Comment (Finrod-Amandil) [inline PokeOneWeb/Frontend/pokeoneweb/src/app/pages/pokemon-list/pokemon-list.component.html]: Tooltip text should be much shorter, i.e. "Bulk = HP + Def. + Sp.D.". Only a quick explanation how the value is calculated should be sufficient.

File: PokeOneWeb/Frontend/pokeoneweb/src/app/pages/pokemon-list/pokemon-list.component.html [hunk 3]
```diff
@@ -1,973 +1,997 @@
...
+                    (click)="sort(column.Spe, -1)"
+                    class="table-header-sort">⮟</div>
+            </th>
+            <th class="col-stat stat-bulk">
+                <div 
+                    [ngClass]="getSortButtonClass(column.Bulk, 1)" 
+                    (click)="sort(column.Bulk, 1)"
+                    class="table-header-sort">⮝</div>
+                <div class="has-tooltip">
+                    <div class="table-header-label">Bulk</div>
```
Comment (Finrod-Amandil) [inline PokeOneWeb/Frontend/pokeoneweb/src/app/pages/pokemon-list/pokemon-list.component.html:822]: Add dotted underline to "Bulk" text to show that there is a tooltip on this text.
CSS:
text-decoration: underline;
text-decoration-style: dotted;
text-decoration-color: var(--light-gray);

**Review by Finrod-Amandil — CHANGES_REQUESTED** [review]

File: PokeOneWeb/Frontend/pokeoneweb/angular.json (hunk 1 above)
Comment (R4ngered) [reply PokeOneWeb/Frontend/pokeoneweb/angular.json]: when starting the angular application with "ng server" I was asked if I want to send analytic data ... I said no :)

File: PokeOneWeb/Frontend/pokeoneweb/src/app/pages/pokemon-list/pokemon-list.component.html (hunk 2 above)
Comment (R4ngered) [reply PokeOneWeb/Frontend/pokeoneweb/src/app/pages/pokemon-list/pokemon-list.component.html]: is fixed now

File: PokeOneWeb/Frontend/pokeoneweb/src/app/pages/pokemon-list/pokemon-list.component.html (hunk 3 above)
Comment (R4ngered) [reply PokeOneWeb/Frontend/pokeoneweb/src/app/pages/pokemon-list/pokemon-list.component.html:822]: has been added to css now

**Review by Finrod-Amandil — APPROVED** [review]

**Pull request merged**